core/vm/memory_table.go

```diff
@@ -48,6 +48,10 @@ func memoryMStore(stack *Stack) (uint64, bool) {
 	return calcMemSize64WithUint(stack.Back(0), 32)
 }
 
+func memoryMemoryCopy(stack *Stack) (uint64, bool) {
+	return calcMemSize64(max(stack.Back(0), stack.Back(1)), stack.Back(2))
```

Comment:
there's no `max` keyword in go. 
Need to do something like: 
```golang
    max := stack.Back(0)
    if stack.Back(1).Gt(max){
        max = steack.Back(1)
    }
    return calcMemSize(max, stack.Back(2))
```

Reply:
fixed in 0be47e9f7a302616c82bdbecf146d1a5b2b6d78f